packages/react/src/useMatomo.ts

```diff
@@ -9,24 +10,43 @@ import {
 } from './types'
 import useOutboundClickListener from './utils/useOutboundClickListener'
 
-function useMatomo() {
+function useMatomo(): {
+  trackPageView: (
+    params: TrackPageViewParams,
+  ) => MatomoTracker['trackPageView'] | void
+  trackSiteSearch: (
+    params: TrackSiteSearchParams,
+  ) => MatomoTracker['trackSiteSearch'] | void
+  trackLink: (params: TrackLinkParams) => MatomoTracker['trackLink'] | void
+  trackEvent: (params: TrackEventParams) => MatomoTracker['trackEvent'] | void
+  trackEvents: () => MatomoTracker['trackEvents'] | void
+  enableLinkTracking: () => void
+} {
   const instance: MatomoInstance = React.useContext(MatomoContext)
```

Comment:
Let's check the presence of the instance once instead of checking it in the methods below. The methods returned by this hook should not have a void return type.